frontend/app/src/main/java/com/example/speechbuddy/di/DatabaseModule.kt

```diff
@@ -44,6 +45,11 @@ class DatabaseModule {
         return appDatabase.categoryDao()
     }
 
+    @Provides
```

Comment:
```suggestion
    @Singleton
    @Provides
```
여기 Singleton 추가해야할 것 같아요!!ㅎㅎ

Reply:
넵넵 추가했습니다!